templates/snyk-ecr.template.yaml

```diff
@@ -36,9 +36,8 @@ Metadata:
 Parameters:
   SnykExternalId:
     Description: Locate the organization ID by logging in to https://app.snyk.io and navigating to Settings.
-    Type: String
-    AllowedPattern: '[a-z0-9-]{36}'
-    MinLength: 1
+    Type: CommaDelimitedList
+    Default: "abcd1234-abcd-1234-abcd-1234abcd1234, abcd1234-abcd-1234-abcd-1234abcd1234"
```

Comment:
We don't put 'dummy' data into the Default of a parameter. If this needs to be filled in, it needs to be left blank, and then the description and deployment guide need to be updated accordingly with instructions for multiple entries.

Reply:
Thanks @dowen12 , that makes sense. I removed the default and updated the description in [b226cb6](https://github.com/aws-quickstart/quickstart-snyk-security/pull/32/commits/b226cb6febcbe99e246a1c0735c5fb1c05124565).

The [deployment guide](https://aws-quickstart.github.io/quickstart-snyk-security/#_deployment_steps) is generic and doesn't describe each parameter, so no action needed?

Reply:
For reference, its in the guide [here](https://aws-quickstart.github.io/quickstart-snyk-security/#_prepare_your_snyk_account).
I am happy with the updated description on these to address the multiple entries.